solaris/solaris11/generate_wazuh_packages.sh

```diff
@@ -160,12 +163,24 @@ create_package() {
     pkgsend generate ${install_path} | pkgfmt > wazuh-agent.p5m.1
     python solaris_fix.py -t SPECS/template_agent_${VERSION}.json -p wazuh-agent.p5m.1 # Fix p5m.1 file
     mv wazuh-agent.p5m.1.aux.fixed wazuh-agent.p5m.1
-
     # Add the preserve=install-only tag to the configuration files
     for file in etc/ossec.conf etc/local_internal_options.conf etc/client.keys; do
         sed "s:file $file.*:& preserve=install-only:"  wazuh-agent.p5m.1 > wazuh-agent.p5m.1.aux_sed
         mv wazuh-agent.p5m.1.aux_sed wazuh-agent.p5m.1
     done
+
+    if [ "${major_version}" -ge "4" ]; then
+        if [ "${minor_version}" -ge "2" ]; then
```

Comment:
Where create we `/var/ossec/logs/wazuh` files?

Reply:
In this issue wazuh/wazuh#6885, `var/ossec/logs/ossec `and `var/ossec/queue/ossec` were renamed to `var/ossec/logs/wazuh` and `var/ossec/queue/sockets` respectively, `/var/ossec/logs/wazuh `is ccreated at compilation time.